Gruntfile.js

```diff
@@ -0,0 +1,286 @@
+/*
+ * GPII Chrome Extension for Google Chrome
+ *
+ * Copyright 2016 RtF-US
+ * Copyright 2017 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this license.
+ *
+ * You may obtain a copy of the license at
+ * https://github.com/GPII/gpii-chrome-extension/blob/master/LICENSE.txt
+ */
+
+ /* global module */
+
+"use strict";
+
+module.exports = function (grunt) {
+    grunt.loadNpmTasks("grunt-contrib-clean");
+    grunt.loadNpmTasks("grunt-contrib-copy");
+    grunt.loadNpmTasks("fluid-grunt-eslint");
+    grunt.loadNpmTasks("grunt-contrib-uglify");
+    grunt.loadNpmTasks("grunt-contrib-stylus");
+    grunt.loadNpmTasks("grunt-crx");
+    grunt.loadNpmTasks("grunt-jsonlint");
+
+    var files = {
+        extensionLib: [
+            "node_modules/infusion/src/framework/core/js/jquery.standalone.js",
+            "node_modules/infusion/src/framework/core/js/Fluid.js",
+            "node_modules/infusion/src/framework/core/js/FluidDebugging.js",
+            "node_modules/infusion/src/framework/core/js/FluidIoC.js",
+            "node_modules/infusion/src/framework/core/js/DataBinding.js",
+            "node_modules/infusion/src/framework/core/js/ModelTransformation.js",
+            "node_modules/infusion/src/framework/core/js/ModelTransformationTransforms.js"
+        ],
+
+        contentScriptsLib: [
+            "node_modules/infusion/src/lib/jquery/core/js/jquery.js",
+            "node_modules/infusion/src/framework/core/js/Fluid.js",
+            "node_modules/infusion/src/framework/core/js/FluidDocument.js",
+            "node_modules/infusion/src/framework/core/js/FluidIoC.js",
+            "node_modules/infusion/src/framework/core/js/DataBinding.js",
+            "node_modules/infusion/src/framework/core/js/FluidView.js",
+            "node_modules/infusion/src/framework/core/js/ModelTransformation.js",
+            "node_modules/infusion/src/framework/core/js/ModelTransformationTransforms.js",
+            "node_modules/infusion/src/framework/core/js/FluidDOMUtilities.js",
+            "node_modules/infusion/src/framework/core/js/FluidRequests.js",
+            "node_modules/infusion/src/lib/fastXmlPull/js/fastXmlPull.js",
+            "node_modules/infusion/src/framework/renderer/js/fluidParser.js",
+            "node_modules/infusion/src/framework/renderer/js/fluidRenderer.js",
+            "node_modules/infusion/src/framework/renderer/js/RendererUtilities.js",
+            "node_modules/infusion/src/framework/enhancement/js/ContextAwareness.js",
+            "node_modules/infusion/src/framework/enhancement/js/ProgressiveEnhancement.js",
+            "node_modules/infusion/src/components/tableOfContents/js/TableOfContents.js",
+            "node_modules/infusion/src/framework/preferences/js/Enactors.js"
+        ],
+        adjustersLib: [
+            // not concatenating all the individual files because there was an issue when including
```

Comment:
What was this issue? Did we get it written up?

Reply:
I don't recall if I came to a definitive conclusion on the reason one. I sort of put aside to finish up the work. I could take another look into it if you think now is a good time.

Reply:
Perhaps we should at least file a placeholder JIRA that just summarises what we know about the issue so far so we can track it here.

Reply:
I was able to switch back to the individual files. I'm not sure if this is something that was fixed along the way or it was just missing a dependency before.